e2e/cypress/integration/system_console/user_management/users_spec.js

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @system_console
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+import {getAdminAccount} from '../../../support/env';
+
+function apiLogin(username, password) {
+    return cy.request({
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        url: '/api/v4/users/login',
+        method: 'POST',
+        body: {login_id: username, password},
+        failOnStatusCode: false,
+    });
+}
+
+describe('System Console > User Management > Users', () => {
+    let testUser;
+    before(() => {
+        // # Login as new user and visit town-square.
+        cy.apiInitSetup({loginAfter: true}).then(({user}) => {
+            testUser = user;
+            cy.apiLogout();
+            cy.apiAdminLogin();
```

Comment:
Is this necessary since we already do this in `L:37` inside the `beforeEach()`
```suggestion
```

Reply:
I don't think it's necessary anymore. Nice catch :+1: